### PR TITLE
CST-100 Starliner

### PR DESCRIPTION
**Pull request by Capkirk123**
Add the Boeing CST-100 starliner from the CST-100 mod to ROCapsules.

### CODE REVIEW

File: GameData/ROCapsules/PartConfigs/CST/CSTHS.cfg [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+	tags = starliner, boeing, cst100, cst-100, 100
+
+	MODULE
+	{
+		name = ModuleDecouple
+		ejectionForce = 5
+		explosiveNodeID = top
+	}
+
+	MODULE:NEEDS[!DeadlyReentry]
```
Comment (DRVeyl) [inline GameData/ROCapsules/PartConfigs/CST/CSTHS.cfg:58]: Want to just add the heatshield tags and let RO assign the values now?

File: GameData/ROCapsules/PartConfigs/CST/CSTCM.cfg [hunk 2]
```diff
@@ -0,0 +1,392 @@
...
+	
+	MODULE
+	{
+		name = FlagDecal
+		textureQuadName = flagTransform
+	}
+}
+
+//correct internals?
+@PART[ROC-CSTCM]:FOR[RealismOverhaul]
```
Comment (DRVeyl) [inline GameData/ROCapsules/PartConfigs/CST/CSTCM.cfg:219]: :BEFORE or :AFTER but not :FOR anything that isn't ROCapsules

File: GameData/ROCapsules/PartConfigs/CST/CSTCM.cfg [hunk 3]
```diff
@@ -0,0 +1,392 @@
...
+	}
+}
+
+//	==================================================
+//	CST-100 command module.
+
+//	Remote Tech compatibility.
+//	==================================================
+
+@PART[ROC-CSTCM]:FOR[RealismOverhaul]:NEEDS[RemoteTech]
```
Comment (DRVeyl) [inline GameData/ROCapsules/PartConfigs/CST/CSTCM.cfg:240]: :BEFORE or :AFTER but not :FOR

File: GameData/ROCapsules/PartConfigs/CST/CSTCM.cfg [hunk 4]
```diff
@@ -0,0 +1,392 @@
...
+}
+
+//	==================================================
+//	CST-100 command module.
+
+//	TAC Life Support compatibility.
+//	==================================================
+
+@PART[ROC-CSTCM]:FOR[RealismOverhaul]:NEEDS[TacLifeSupport]
+{
```
Comment (DRVeyl) [inline GameData/ROCapsules/PartConfigs/CST/CSTCM.cfg:309]: :BEFORE or :AFTER but not :FOR

File: GameData/ROCapsules/PartConfigs/CST/CSTCM.cfg [hunk 5]
```diff
@@ -0,0 +1,392 @@
...
+		}
+	}
+}
+
+//	==================================================
+//	CST-100 command module.
+
+//	Real Antennas compatibility.
+//	==================================================
+@PART[ROC-CSTCM]:BEFORE[RealAntennas]:NEEDS[RealAntennas]
```
Comment (DRVeyl) [inline GameData/ROCapsules/PartConfigs/CST/CSTCM.cfg:285]: :BEFORE[x]:NEEDS[x] is redundant; BEFORE implies NEEDS. But it's common, so nbd.

**Comment by pap1723** [comment]
@Capkirk123 what is the status of this? I want to create a release of ROCapsules soon and want to get all of this stuff integrated

**Pull request merged**